packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -73,3 +76,24 @@ export function Disclosure({
     setOpen(!currentToggleState);
   }
 }
+
+interface DisclosureTitleProps {
+  /**
+   * Title for the disclosure pane.
+   */
+  readonly title: string | ReactNode | ReactNode[];
+  /**
+   * Size when the title is string.
+   */
+  readonly size: "base" | "large";
+}
+
+const DisclosureTitle = ({ title, size }: DisclosureTitleProps) => {
```

Comment:
This is my mistake, but I think using function declarations would a preferred way (vs. function expressions):

```suggestion
function DisclosureTitle({ title, size }: DisclosureTitleProps) => {
```